src/Network/Packets/ServerToClient/ServerToClientPackets.xml

```diff
@@ -5250,6 +5250,75 @@
         </Field>
       </Fields>
     </Packet>
+    <Packet>
+      <HeaderType>C1HeaderWithSubCode</HeaderType>
+      <Code>BF</Code>
+      <SubCode>51</SubCode>
+      <Name>MuBotUseResponse</Name>
+      <Length>16</Length>
+      <Direction>ServerToClient</Direction>
+      <SentWhen>The server validate (and toggle status) if user can use the bot</SentWhen>
+      <CausedReaction>The client toggle the mu bot status</CausedReaction>
+      <Fields>
+        <Field>
+          <Index>4</Index>
+          <Type>Byte</Type>
+          <Name>IsZen</Name>
```

Comment:
PMSG_MUBOT_USE.Time

Reply:
I tried to use the packet specification described in IGCN files and is not working, then i switch to this approach (from SCF) and works fine.
This byte is used to tell the client that must show the money (specified in packet[8:12]) consumed by mu bot.